ui/lib/components/ParamPageWrapper/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactNode } from 'react'
+import { useParams } from 'react-router-dom'
+
+export interface IParamPageWrapperProps {
+  paramName?: string
+  children: ReactNode
+}
+
+export default function ParamPageWrapper({
+  paramName = 'id',
+  children,
+}: IParamPageWrapperProps) {
+  const paramValue = useParams()[paramName]
```

Comment:
How about simply use the JSON stringify value of `useParams()` as the key? Then no need to specify the param name.

Reply:
Good idea! Let me try it.

Reply:
submit PR in #527